src/middleware/PostQuery.ts

```diff
@@ -0,0 +1,64 @@
+import { Request, Response, NextFunction } from 'express';
+
+/*
+ *  This class will be for handling the transfer of the res.locals.graphqlGate object
+ *  to the webapp's backend
+ */
+
+export default class PostQuery {
+    private gateURI: string;
+
+    constructor(gateURI: string) {
+        this.gateURI = gateURI;
```

Comment:
I think this should be hard coded into the function/class, not instantiated with a constructor. The reason being that this is the same URI for every usage of the gate-logger middleware because every application is logging data to our server. A more important parameter for the constructor function would be tho project ID that this class is sending requests to.

Reply:
The reasoning I had for this is that in our index.ts, we will be connecting all of the middleware into a chain, and will want all of the gateURIs to be the same without having to change the value in multiple middleware files.